### PR TITLE
Add threads options.

### PR DESCRIPTION
**Pull request by GunhouWong**

### CODE REVIEW

**Comment by emretetik96** [comment]
Is there a reason this PR was never merged? Would be helpful to be able to use threads with this library

**Comment by emretetik96** [comment]
@bramp on the above

**Comment by jamerlan** [comment]
ability to specify threads is very useful

File: src/main/java/net/bramp/ffmpeg/builder/FFmpegBuilder.java [hunk 1]
```diff
@@ -150,7 +151,13 @@ public FFmpegBuilder setInput(String filename) {
     return addInput(filename);
   }
 
-  public FFmpegBuilder setFormat(String format) {
+    public FFmpegBuilder setThreads(int threads) {
+        checkArgument(threads != 0, "threads must be greater than zero");
```
Comment (drallgood) [inline src/main/java/net/bramp/ffmpeg/builder/FFmpegBuilder.java:155]: Should check for >0

Comment (bramp) [reply src/main/java/net/bramp/ffmpeg/builder/FFmpegBuilder.java:155]: Good catch. Please make the change.

**Comment by bramp** [comment]
Looks good other than one minor change.

**Pull request closed without merging**